addons/account/models/account_move.py

```diff
@@ -3499,6 +3500,10 @@ def _post(self, soft=True):
                 customer_count[invoice.partner_id] += 1
             elif invoice.is_purchase_document():
                 supplier_count[invoice.partner_id] += 1
+            elif invoice.move_type == 'entry':
+                sale_amls = invoice.line_ids.filtered(lambda line: line.debit and line.partner_id and line.account_id.account_type == 'asset_receivable')
+                for partner, amls in groupby(sale_amls, lambda aml: aml.partner_id):
+                    customer_count[partner] += 1
```

Comment:
Heyyy @moerradi sorry I realized that the groupby isn't even necessary, we could just do 
```suggestion
                for partner in sale_amls.mapped('partner_id')
                    customer_count[partner] += 1
```
Sorry for making you change things again!

Reply:
no problem !
done !